spec/dummy/app/views/application/index.html.erb

```diff
@@ -1,5 +1,12 @@
 <% if params[:index_html] %>
-  <%= include_ember_index_html "my-app" %>
+  <% if params[:csrf] %>
+    <%= include_ember_index_html "my-app" do %>
```

Comment:
This is messy and certainly not scalable for more combinations of invocations, but whenever I tried to dedicate each helper with its own ERB view the tests would inexplicably fail.